app/javascript/components/DualPaneMapper/DualPaneMapperCount.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 
 const DualPaneMapperCount = ({ selectedItems, totalItems }) => (
   <div className="dual-pane-mapper-count">
-    {selectedItems} of {totalItems} items selected
+    {selectedItems} {__('of')} {totalItems} {__('items selected')}
```

Comment:
@michaelkro Can you use `sprintf` here in order to keep the whole sentence intact.

Something like this -
```js
__(sprintf('%s of %s items selected', {selectedItems}, {totalItems}));
```

Whole sentences work better with translators.